README.md

```diff
@@ -84,6 +84,7 @@ Parameters for the app are stored in [environment variables](http://man7.org/lin
 | `VUE_APP_FHIR_RESOURCES` | For specifying which FHIR resource(s) to load for the selected patient. | `Condition,Procedure,Observation,QuestionnaireResponse` |
 | `VUE_APP_AUTH_SCOPES` | For allowing the app to specify the delegation of a specific set of access rights via launch context. see [App Launch: Scopes and Launch Context](https://build.fhir.org/ig/HL7/smart-app-launch/scopes-and-launch-context.html) | `profile roles email patient/*.read openid fhirUser patient/QuestionnaireResponse.write` |
 | `VUE_APP_CONF_API_URL` | URL from which the app can retrieve launch context, i.e. launching as a confidential client | ''
+| `VUE_APP_PROJECT_ID` | Project ID that uniquely identifies the project.  The app will display project specific logo and theme based on the ID | 'DEM'
```

Comment:
@achen2401  is it too late to change all "DEM" references to "DCW"?

Reply:
@mcjustin I updated all "DEM" references to "DCW".  Just want to note that, do we need to make the same change for the [environments repo](https://github.com/uwcirg/dem-environments/tree/main/dev) as well?

Reply:
Thanks @achen2401 for doing that, and for flagging environments. I'll create a PR over there FYI @ivan-c , might as well nip it in the bud